flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/converters/XMLToRecordConverter.java

```diff
@@ -103,17 +103,20 @@ public final class XMLToRecordConverter {
     private final io.ballerina.projects.Document document;
     private final Path filePath;
     private final TypesManager typesManager;
+    private final Map<String, String> importStatements;
 
     public XMLToRecordConverter(Project project, io.ballerina.projects.Document document, Path filePath,
                                 TypesManager typesManager) {
         this.project = project;
         this.document = document;
         this.filePath = filePath;
         this.typesManager = typesManager;
+        this.importStatements = new HashMap<>();
```

Comment:
We are putting only one value here, so do we need to maintain a map?

Reply:
Addressed via 9220d002045e522526d52ac3f3767ea3e9675a09